### ISSUE TITLE
[TGL][SDW] runtime PM always in active on TGL chromebook/SDW mode

### ISSUE THREAD

**Issue by YvonneYang2** [thesofproject/linux#2162]
**Describe the bug**
runtime PM always in active on TGL chromebook/SDW mode. 

**To Reproduce**
Check runtime PM status when sof is idle for more than 3 seconds.
```
$cat /sys/bus/pci/devices/0000\:00\:1f.3/power/runtime_status
```
**Reproduction Rate**
10/10

**Expected behavior**
runtime PM will be suspended after sof is idle for more than 3 seconds.

**Environment**
Kernel:https://github.com/thesofproject/linux/commits/topic/sof-dev commit:5cb79ee6f
Revert regression on USB-C 6a7c533d4a1854f54901a065d8c672e890400d8a
Revert regression on PINCTRL cd0a32371db73d0b50536a7ca4f036abddff0d1d+ #2062  + hack commit https://github.com/thesofproject/linux/commit/6769df9d4d48cfae008a8192aa5b723bc8edb804
FW:: https://github.com/thesofproject/sof/tree/tgl-006-drop-stable plus https://github.com/thesofproject/sof/pull/3006
TPLG: sof-tgl-sdw-max98373-rt5682-4ch.tplg rename to sof-tgl-sdw-max98373-rt5682.tplg
Platform: TGL SDW chromebook with rt5682-max98373

[logger.log](https://github.com/thesofproject/linux/files/4716717/logger.log)
[dmesg.log](https://github.com/thesofproject/linux/files/4716718/dmesg.log)

**Comment by YvonneYang2** [comment]
Set them 
amixer -c0 cset name='Left Spk Switch' off
amixer -c0 cset name='Right Spk Switch' off
Then check the pm runtime status, still in active status.

**Comment by naveen-manohar** [comment]
Observe that master-Controller-1, that connects to max98373 slaves is always active. (Where-as Controller-0 connecting to RT5682 goes to suspend when not in use.)
cat /sys/bus/soundwire/devices/sdw-master-1/sdw\:1\:19f\:8373\:0\:3/power/runtime_status
active
cat /sys/bus/soundwire/devices/sdw-master-1/sdw\:1\:19f\:8373\:0\:7/power/runtime_status
active

**Comment by RanderWang** [comment]
The issue is different with I2S mode. And the patch https://github.com/thesofproject/linux/pull/2123/commits/8947163714a57dbce92caaa637778db534c30392 doesn't work because user mode API will enable spk pin after card probe on sdw mode.

**Comment by RanderWang** [comment]
And according to https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/codecs/max98373.c#L886,  the codec can not be suspended

**Comment by RanderWang** [comment]
Fixed, I will submit a PR to fix it tomorrow after the last ACK from maximum developer.

**Comment by YvonneYang2** [comment]
Issue is fixed by #2201 and no side effect found.
